tests/foreman/api/test_organization.py

```diff
@@ -478,69 +341,3 @@ def test_negative_update(self):
                         id=self.organization.id,
                         **attrs
                     ).update(attrs.keys())
-
-    @tier2
-    @upgrade
-    @skip_if_bug_open('bugzilla', 1103157)
-    def test_verify_bugzilla_1103157(self):
```

Comment:
@pondrejk This test looks good. Why we re removing it? Is it already covered somewhere?

Reply:
yeah I was a bit hesitant about this one, adding multiple compute resources and checking there is no conflict is test_positive_add_and_remove_compresources in cli. This test does it non-admin user, but I didn't feel it justifies a separate test case. Anyway, if there's a strong opinion to keep it in, I'm ok with that.

Reply:
I am ok with it if same scenario without non-admin user is covered in another test. Thanks.